cl8/users/models.py

```diff
@@ -284,6 +286,23 @@ def generate_invite_mail(self):
 
         return {"text": rendered_invite_txt, "html": rendered_invite_html}
 
+    def vcard(self) -> vobject.vCard:
+        properties = {
```

Comment:
We could add additional properties in the future, but I think this is a good starter set. The [vCard Wikipedia page](https://en.wikipedia.org/wiki/VCard#Properties) has a good list of these properties, as does [the spec](https://datatracker.ietf.org/doc/html/rfc6350).